server/database/queries/visits_all_export.js

```diff
@@ -0,0 +1,12 @@
+const getAllVisitsQuery = `
```

Comment:
Isn't this basically identical to `visitors_all` except for the `ORDER BY`? Can you not just use the `visitors_all` endpoint instead and then sort on the client?

Reply:
I was going to do this but I needed the visitors name rather than their id as is returned by visitors_all.

Reply:
is it possible to add the `visitor_name` to the other query without breaking anything?

Reply:
Well, currently we show the users id rather than name on visits data page. Would it make more sense to actually display the name? The only reason I need the name for the csv is for clarity rather than an arbitrary number; which holds true for the page as well. In that case I would be able to use the exact same query.

Reply:
No, what I mean is, send them both, and just pick what is needed for either page

Reply:
Do you mean delete the values `visitor_id: 5` or `visitor_name: Jeremy` from the array of objects in the route?

Reply:
No, just send both from the backend, and just don't pick the values that aren't necessary on the client.